pkgs/development/coq-modules/coq-bits/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, mkCoqDerivation, coq, mathcomp, version ? null }:
+{ lib, mkCoqDerivation, coq, ssreflect, mathcomp-algebra, version ? null }:
```

Comment:
It looks to me like ssreflect isn't used anymore, so it shouldn't need to be listed here.